schema/google/showcase/v1beta1/compliance.proto

```diff
@@ -96,6 +96,29 @@ service Compliance {
     };
   }
 
+  // This method returns an unknown value for a non-optional enum field. Client libraries should
+  // either error gracefully (not crash), ignore the value, or, depending on the API, set it to the
+  // zero default.
+  //
+  // This only works over REST currently. Requests over gRPC will simply echo the request.
+  rpc RepeatWithUnknownEnum(RepeatRequest) returns (RepeatResponse) {
+    option (google.api.http) = {
+      post: "/v1beta1/repeat:invalidenum"
+      body: "*"
+    };
+  }
+
+  // This method returns an unknown value for an optional enum field. Client libraries should either
+  // error gracefully (not crash), ignore the value, or, depending on the API, set it to the zero
+  // default.
+  //
+  // This only works over REST currently. Requests over gRPC will simply echo the request.
+  rpc RepeatWithUnknownOptionalEnum(RepeatRequest) returns (RepeatResponse) {
```

Comment:
I've been thinking about this (and sidetracked by other issues, sorry). We could keep just the previously existing RPCs and add a top-level field to `RepeatRequest` to generate the unknown enum. In fact, I would change `RepeatRequest.server_verify` to be an enum like `server_behavior` that could be `verify`, or `generate_unknown_enum`, or something else in the future.

Not too concerned about that being a breaking change because (a) most generators are not yet using the compliance suite, and (b) the compliance suite is meant to be parsed form this repo and then used to issue requests, without the client code modifying the request.

WDYT?

Reply:
Yeah that sounds great to me. Thanks for considering this.